CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v1.24.0 (2024-09-27)
+
+## Build Changes
+
+### Twoliter
+* Update bottlerocket-core-kit to v2.8.1 ([#4222])
```

Comment:
Can we break out two points that are important here: the nvidia container toolkit update and the open gpu change. Can we add those to a section at the top that is "Release Highlights" so that it is clear what is changing here?